src/libraries/System.Private.DataContractSerialization/src/System/Runtime/Serialization/Globals.cs

```diff
@@ -314,21 +314,25 @@ internal static Type TypeOfHashtable
         internal static Type TypeOfDBNull =>
             s_typeOfDBNull ??= typeof(DBNull);
 
-        private static Uri? s_dataContractXsdBaseNamespaceUri;
-        internal static Uri DataContractXsdBaseNamespaceUri =>
-            s_dataContractXsdBaseNamespaceUri ??= new Uri(DataContractXsdBaseNamespace);
+        [DynamicallyAccessedMembers(DynamicallyAccessedMemberTypes.NonPublicFields)]
+        private static Type? s_typeOfSchemaDefinedType;
+        [DynamicallyAccessedMembers(DynamicallyAccessedMemberTypes.NonPublicFields)]
+        internal static Type TypeOfSchemaDefinedType =>
+            s_typeOfSchemaDefinedType ??= typeof(SchemaDefinedType);
 
-        private static readonly Type? s_typeOfScriptObject;
+        [DynamicallyAccessedMembers(DynamicallyAccessedMemberTypes.NonPublicFields)]
+        private static Type? s_typeOfSchemaDefinedEnum;
+        [DynamicallyAccessedMembers(DynamicallyAccessedMemberTypes.NonPublicFields)]
+        internal static Type TypeOfSchemaDefinedEnum =>
+            s_typeOfSchemaDefinedEnum ??= typeof(SchemaDefinedEnum);
 
-        [RequiresUnreferencedCode(DataContract.SerializerTrimmerWarning)]
-        internal static ClassDataContract CreateScriptObjectClassDataContract()
-        {
-            Debug.Assert(s_typeOfScriptObject != null);
-            return new ClassDataContract(s_typeOfScriptObject);
-        }
+        private static MemberInfo? s_schemaMemberInfoPlaceholder;
+        internal static MemberInfo SchemaMemberInfoPlaceholder =>
+            s_schemaMemberInfoPlaceholder ??= TypeOfSchemaDefinedType.GetField("_xmlName", BindingFlags.NonPublic | BindingFlags.Instance)!;
 
```

Comment:
I think you can use nameof(SchemaDefinedType._xmlName) and it will evaluate to "_xmlName";